Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -97,8 +97,7 @@ void VideoConfig::Load(const std::string& ini_file)
 
 	IniFile::Section* hacks = iniFile.GetOrCreateSection("Hacks");
 	hacks->Get("EFBAccessEnable", &bEFBAccessEnable, true);
-	hacks->Get("EFBCopyEnable", &bEFBCopyEnable, true);
-	hacks->Get("EFBToTextureEnable", &bCopyEFBToTexture, true);
+	hacks->Get("EFBToTextureEnable", &bSkipEfbCopy2Ram, true);
```

Comment:
I'll rename this ini option when the current open ini PR is merged. Too many conflicts right now.